From e358a47782ec0efb7eb8b980e49964d015620cd4 Mon Sep 17 00:00:00 2001
From: Roman Li <Roman.Li@amd.com>
Date: Mon, 14 Aug 2017 17:35:08 -0400
Subject: [PATCH 3445/5855] drm/amd/display: Fix FBC compilation

- Fixing compilation with ENABLE_FBC flag
  which was broken after flattening core_dc to dc

Signed-off-by: Roman Li <Roman.Li@amd.com>
Reviewed-by: Jerry Zuo <Jerry.Zuo@amd.com>
Acked-by: Harry Wentland <Harry.Wentland@amd.com>
---
 drivers/gpu/drm/amd/display/dc/dc.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/amd/display/dc/dc.h b/drivers/gpu/drm/amd/display/dc/dc.h
index 025392d..bbd2240 100644
--- a/drivers/gpu/drm/amd/display/dc/dc.h
+++ b/drivers/gpu/drm/amd/display/dc/dc.h
@@ -35,6 +35,7 @@
 #include <inc/hw/opp.h>
 
 #include "inc/hw_sequencer.h"
+#include "inc/compressor.h"
 #include "dml/display_mode_lib.h"
 
 
-- 
2.7.4

